src/components/Navbar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import styles from 'styles/Nav.module.css';
+import logo from './image/logo.png';
+
+const Navbar = () => (
+  <nav className={styles.navBar}>
+    <div className={styles.logo}>
+      <NavLink to="/" className={styles.link}>
+        <img src={logo} alt="travelers logo " />
+        <h1>Space Rockets</h1>
+      </NavLink>
+    </div>
+
+    <ul className={styles.navLinks}>
+      <li className={styles.links}>
+        <NavLink to="/" className={styles.link}>
+          Rockets
+        </NavLink>
+      </li>
+      <li>
+        <NavLink to="/mission" className={styles.link}>
```

Comment:
- [ ] A great way to keep the active page link underlined link underlined even when the link is not in focus is to use the ` style={({ isActive }) => ({` that comes with `NavLink`. 

## EXAMPLE
```
        <NavLink
          to="/mission"
          style={({ isActive }) => ({
            textDecoration: isActive ? 'underline' : '#545e6f',
          })}
          className={styles.link}
        >
          Missons
        </NavLink>
```

This will ensure that the NavLink is still underlined even when the user clicks outside the NavLink after clicking the NavLink





https://user-images.githubusercontent.com/52934128/224384801-396a8451-adc3-4a0a-aa60-1a345d4c51b6.mp4